docs/pages/access-controls/sso/azuread.mdx

```diff
@@ -204,11 +206,11 @@ should enable token encryption, read the Azure AD
 
 ### Set up Teleport token encryption
 
-Start with generating a public/private key and a certificate. You will set up the public
-certificate with Azure AD and the private key with Teleport.
+Start with generating a public/private key and a certificate. You will set up
+the public certificate with Azure AD and the private key with Teleport.
 
 ```code
-$ openssl req  -nodes -new -x509  -keyout server.key -out server.cer
+$ openssl req -nodes -new -x509 -keyout server.key -out server.cer
 ```
 
```

Comment:
It feels like something is missing here. What if I'm not modifying an existing connector?
"You will notice that Teleport has generated a `signing_key_pair`. This key pair
is used to sign responses."
Where am I noticing this? in the YAML file I generated if I was modifying an existing connector?

Reply:
(FYI It took me a minute to realize what lines you were referring to)

Same as above, I'm not an SME on this section to answer your questions.